.gitignore

```diff
@@ -33,6 +33,7 @@ _site/
 node_modules/
 dist/
 react/
+/svelte/
```

Comment:
Is there something wrong with the pattern that we were following, with the / before the folder name?
Is there a problem by using `svelte/` instead of `/svelte/`?